ieee.csl

```diff
@@ -353,6 +392,11 @@
             <text macro="publisher"/>
             <text macro="issued"/>
             <text macro="page"/>
+            <choose>
+              <if variable="URL">
```

Comment:
No need to test for URL here (and if testing should be for URL or DOI) since we do this in the macro.